Marlin/ultralcd.cpp

```diff
@@ -3460,17 +3457,15 @@ void lcd_update() {
   #if ENABLED(ULTIPANEL)
     static millis_t return_to_status_ms = 0;
     manage_manual_move();
-  #endif
 
-  lcd_buttons_update();
+    lcd_buttons_update();
```

Comment:
main dev does not use an adc_keypad / lcd .. afaik the part is needed for that display to work. i am not sure about the need of another ifdef though.